src/ape/managers/chain.py

```diff
@@ -504,6 +505,44 @@ def __getitem__(self, address: AddressType) -> ContractType:
 
         return contract_type
 
+    def get_all(
```

Comment:
Not a huge fan of this name

Reply:
Ok, give me an alternative

Reply:
Or at least tell me why you don't like it

Reply:
Sorry, it wasn't very descriptive

Reply:
`chain.contracts.get_multiple(addresses)` seems just as descriptive as `chain.contracts[address]` or `chain.contracts.get(address)`

Reply:
Sure, that works